centraldogma/src/test/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplierIntegrationTest.java

```diff
@@ -119,6 +120,11 @@ public void testCDIntegration() throws InterruptedException {
 
         latch.await();
         assertEquals(20, prop.value().intValue());
+
+        IntStream.range(0, 10000)
+                 .mapToObj(i -> CONFIG_PARTITION_CONCURRENCY)
+                 .map(supplier::getProperty);
+        assertEquals(1, supplier.getCachedProperties().size());
```

Comment:
To check cache is working properly, I think it's better all property returned from `supplier.getProperty` are same reference, rather than checking the internal hashmap's content

Reply:
Fair enough. Fixed